bundles/org.openhab.ui/web/src/pages/settings/items/item-details.vue

```diff
@@ -184,7 +184,13 @@ export default {
       this.$store.dispatch('stopTrackingStates')
     },
     load () {
-      this.$oh.api.get(`/rest/items/${this.itemName}?metadata=semantics,${MetadataNamespaces.map((n) => n.name).join(',')}`).then((data) => {
+      this.$oh.api.get(`/rest/items/${this.itemName}?metadata=.+`).then((data) => {
```

Comment:
This fetch is also in the model page. It doesn't work there (search for MetadataNamespaces across the project):

![image](https://user-images.githubusercontent.com/2004147/213591745-10014b4f-b35d-4285-9bca-945396a853d8.png)